hedyorg/hedy
# [FIX, LANGUAGE] Disallow 'random' indents

Long overdue improvement of the indenter. Fixes #3585.

~This is still WIP, because I also want to do #3291 while I am at it!~ Never mind I will do that seperately cause it is so much work.

**How to test**

Go to level 8 or higher, place spaces before a line and observe you get a nice error message, f.e.:

```
      print 'hello'
```
